### DISCUSSION TITLE
v5 Validation / indicative use

### DISCUSSION THREAD

**Discussion by skrenek** [adonisjs/core#3888] [General]
From what I can see, the Indicative validation library (from pre-v5) is in "sunset" mode, and any development related to it is being continued in @adonisjs/validator.  Is this correct?

I have a project outside of AdonisJS that I'd love to use indicative in, but don't want to rely on it if it's pretty much dead.

The new validator project doesn't look to be as usable outside of Adonis as Indicative was/is.

**Comment by thetutlage** [comment]
Yup, indicative is not active. All the work has been moved to `@adonisjs/validator`. It is hard for me to maintain too many agnostic packages

**Reply by skrenek** [reply]
Completely understandable.  Thanks for all your work!
